examples/demo-rollup/src/test_rpc.rs

```diff
@@ -0,0 +1,167 @@
+use std::net::SocketAddr;
+
+use curl::easy::{Easy2, Handler, List, WriteError};
```

Comment:
Is there a particular reason why you went with `curl`? `reqwest` is much higher level framework and "standard" in rust?